linera-execution/src/policy.rs

```diff
@@ -33,10 +56,28 @@ impl Pricing {
         Ok(self.messages.try_mul(size)?)
     }
 
-    pub fn storage_price(&self, data: &impl Serialize) -> Result<Amount, PricingError> {
+    pub fn storage_num_reads_price(&self, size: &u64) -> Result<Amount, PricingError> {
+        let size = *size as u128;
+        Ok(self.storage_num_reads.try_mul(size)?)
```

Comment:
Is this still being multiplied by size? Or does "size" mean "number of operations" here, as opposed to in the other methods? Then I'd call it "count".

Reply:
Then I renamed as `count`. Thanks.